src/CMakeLists.txt

```diff
@@ -338,8 +338,8 @@ find_package(Qt5 5.6 REQUIRED COMPONENTS Core Multimedia Network OpenGL UiTools
 # This is not present before Qt 5.7:
```

Comment:
You need to increase the minimum version to find in the `find_package(Qt5 ....)` on line 336 to `5.7`...!

Reply:
Updated.